Makefile

```diff
@@ -73,11 +77,8 @@ docker-push: docker-build
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)
 
 clean:
-	rm -rf .tmp
-	rm -rf phase3/.tmp
-	rm -rf phase1/gce/.tmp
-	rm -rf phase1/azure/.tmp
-	rm -rf phase1/vsphere/.tmp
+	rm -rf phase3/*/.tmp/
+	rm -rf phase1/*/*/
```

Comment:
@pipejakob, ignore my above message. there is a simple trick to achieve what we wanted.